hms-app/src/components/card-details/CategoryDetails.tsx

```diff
@@ -70,7 +71,10 @@ export default function CategoryDetails(props: IProps) {
       </Text>
       <Text className={classes.title}>Title: {categoryData.title}</Text>
       {!isLoading && (
-        <Button color={'red'} onClick={() => deleteSelectedCategory()}>
+        <Button
+          style={{ backgroundColor: LEAVE_BUTTON_COLOR }}
```

Comment:
should probably be `DELETE_BUTTON_COLOR` (I know it doesn't exist yet but maybe should)

Reply:
or I guess this is more... a confirm button :thinking:

Reply:
basically my point is that if you have colors with names specific to where they are used then that color should only be used there.  And then when you have other places you should have other named colors

Reply:
what you think about RED_BUTTON as name?

Reply:
or split between leave and delete button?